public/docs/_examples/component-styles/dart/lib/hero_app_component.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:angular2/core.dart';
+import 'hero.dart';
+import 'hero_app_main_component.dart';
+
+// #docregion
+@Component(
+  selector: 'hero-app',
+  template: '''
+    <h1>Tour of Heroes</h1>
+    <hero-app-main [hero]="hero"></hero-app-main>''',
+  styles: const ['h1 { font-weight: normal; }'],
+  directives: const [HeroAppMainComponent])
+// #enddocregion
+class HeroAppComponent {
+  var hero = new Hero('Human Torch', [
```

Comment:
`final Hero` instead of `var`, it gives you type safety and later don't put something that's not a `Hero` in your `hero` variable.